readthedocs/profiles/views.py

```diff
@@ -211,7 +210,7 @@ def account_advertising(request):
     )
 
 
-class TokenMixin:
```

Comment:
Is this the actual bugfix here?

Reply:
Yeah, otherwise an anonymous user could hit that page, it was returning 500. There was a discussion on slack, sorry for the lack of context.